### ISSUE TITLE
Device not found, check IP address

### ISSUE THREAD

**Issue by xX-Nexus-Xx** [andreypopov/node-red-contrib-miio-roborock#42]
Since a few weeks the node is not detecting my Roborock S7 anymore. The respective IP adress is found via PING from Node-Red. The account details and TOKEN are identical to what is used in HA (it is working there)

![image](https://user-images.githubusercontent.com/2195190/151489532-04e3246a-9d19-4bae-9d76-890307c3a859.png)

This was working a few weeks back !

Anybody having the same issues? or even some recommendation on how to solve or investigate?

thx a lot

**Comment by andreypopov** [comment]
There were no updates for 2 months. 
Something is with your network connection, try to extract token and set it manually.

**Comment by xX-Nexus-Xx** [comment]
I'm aware there was no update to the node neither did I changed anything on my network :(

I already extracted the token manually and configured accordingly. HA and Node-red are running on same device. It does work in HA but not in Node-Red :(

I have tried so far:
- I used a token-extractor to get the S7 token an applied manually to Node-Red node and HA-Xiaomi config
- ping the S7 (192.168.2.22) from within Node-Red (192.168.2.202) => working fine
- Using HA-Xiaomi integration (192.168.2.202) with same TOKEN and User-ID/PWD => works fine

Any idea on what i can test or investigate?
